src/main/kotlin/bingsoochef/bingsoochef/bingsoo/presentation/BingsooSwaggerInterface.kt

```diff
@@ -0,0 +1,51 @@
+package bingsoochef.bingsoochef.bingsoo.presentation
+
+import bingsoochef.bingsoochef.bingsoo.presentation.req.CreateBingsooRequest
+import bingsoochef.bingsoochef.bingsoo.presentation.req.UpdateBingsooRequest
+import bingsoochef.bingsoochef.bingsoo.presentation.res.BingsooDetailResponse
+import bingsoochef.bingsoochef.bingsoo.presentation.res.BingsooResponse
+import io.swagger.v3.oas.annotations.Operation
+import io.swagger.v3.oas.annotations.media.ArraySchema
+import io.swagger.v3.oas.annotations.media.Content
+import io.swagger.v3.oas.annotations.media.Schema
+import io.swagger.v3.oas.annotations.responses.ApiResponse
+import io.swagger.v3.oas.annotations.responses.ApiResponses
+import io.swagger.v3.oas.annotations.tags.Tag
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.RequestBody
+
+@Tag(name = "Bingsoo", description = "빙수 API")
+interface BingsooSwaggerInterface {
```

Comment:
API도 여러 버전이 추후 존재 할 수 있어서 Swagger 네이밍은 굳이 안써도 될 것 같은데 어떻게 생각하시나요?
```suggestion
interface BongsooControllerInterface {
```